### ISSUE TITLE
iOS build failing

### ISSUE THREAD

**Issue by shreyasb0912** [facebook/react-native#25250]
iOS build is failing with following error:

  In file included from /Users/cci/Work/React Native/My 
  Projects/helloworld/node_modules/react- 
  native/Libraries/Settings/RCTSettingsManager.m:8:
  /Users/cci/Work/React Native/My Projects/helloworld/node_modules/react- 
  native/Libraries/Settings/RCTSettingsManager.h:10:9: fatal error: 
  'React/RCTBridgeModule.h' file not found
   #import <React/RCTBridgeModule.h>
        ^~~~~~~~~~~~~~~~~~~~~~~~~
   1 error generated.

   info 

   error Failed to build iOS project. We ran "xcodebuild" command but it exited with error 
   code 
   65. To debug build logs further, consider building your app with Xcode.app, by opening 
   helloworld.xcodeproj

React Native version:

  info 
  React Native Environment Info:
    System:
      OS: macOS Sierra 10.12.6
      CPU: (4) x64 Intel(R) Core(TM) i3 CPU         540  @ 3.07GHz
      Memory: 227.75 MB / 4.00 GB
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 10.16.0 - /usr/local/bin/node
      npm: 4.6.1 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    SDKs:
      iOS SDK:
        Platforms: iOS 11.2, macOS 10.13, tvOS 11.2, watchOS 4.2
    IDEs:
      Xcode: 9.2/9C40b - /usr/bin/xcodebuild
    npmPackages:
      react: 16.8.3 => 16.8.3 
      react-native: 0.59.9 => 0.59.9 
    npmGlobalPackages:
      react-native-cli: 2.0.1

## Steps To Reproduce

1. Run react-native run-ios

**Comment by shreyasb0912** [comment]
Now getting error as follow:

fatal error: 'UIKit/UIUserActivity.h' file not found

**Comment by jerearaujo03** [comment]
Same problem here

**Comment by hramos** [comment]
Did this start happening spontaneously, or is it a new issue that appeared after upgrading your project to React Native 0.59? If the latter, I would advise trying again with Xcode 10.2.1 and the iOS 12 SDK, as that is what we're currently using in our test target.

**Comment by shreyasb0912** [comment]
Well, I have just started learning React Native.

I followed the official documentation of React Native to create a sample project and then running it on iOS Simulator. I tried running the project via Xcode as well but the build is failing. I also followed the steps mentioned in other threads regarding the same issue, but still no luck.

I have the latest version Xcode and React Native installed on my Mac machine.
